doc/api/cli.md

```diff
@@ -377,6 +505,20 @@ added:
 
 Enable experimental support for the `https:` protocol in `import` specifiers.
 
+### `--experimental-permission`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Enable the Permission Model for current process. When enabled, the
+following permissions are restricted:
+
+* File System - manageable through
+  \[`--allow-fs-read`]\[],\[`allow-fs-write`]\[] flags
+* Child Process - manageable through \[`--allow-child-process`]\[] flag
+* Worker Threads - manageable through \[`--allow-worker`]\[] flag
```

Comment:
What about native add-ons? Those should also be disabled by default

Reply:
Actually, I raised my concerns about a flag `--allow-addons` [here](https://github.com/nodejs/node/pull/44004#issuecomment-1354819672).

Also, as previously mentioned by @richardlau, we do have a flag `--no-addons` (see https://github.com/nodejs/node/pull/44004#issuecomment-1354919977). Perhaps we can just mention it in `permissions.md`?

Reply:
We can also enable this flag (--no-addons) when using the permission model unless a `--allow-addons` is passed. But, I think it might be confusing for the users.

Reply:
I don't think it would be confusing as long as we document it accordingly, but it doesn't have to happen in this PR.

Reply:
I really believe we need to disable native addons by default when permissions are enabled, even if there currently is no way to re-enable them right away. It is too easy to bypass any permissions model using a native addon and allowing them opens a major hole in the model.

Reply:
I was considering doing it in a separate PR. Anyway, I think I'll follow the approach: `--no-addons` by default unless `--allow-addons` is passed. Do you believe it should land in this PR?

Reply:
Yes I do. At least the setting --no-addons by default bit. The --allow-addons can be added separately